README.md

```diff
@@ -1,104 +1,166 @@
 
-### **Setup via a JSON5 configuration file**
+   ```sh
+   cargo install cargo-run-bin
+   cargo bin zenohd --config EXAMPLE_CONFIG.json5
+   ```  
 
-  - Create a `zenoh.json5` configuration file containing for example:
-    ```json5
-    {
-      plugins: {
-        // configuration of "storage_manager" plugin:
-        remote_api: {
-          "websocket_port": "10000",
-          // secure_websocket configuration is optional
-          "secure_websocket": {
-                "certificate_path" : "/path/to/certificate",
-                "private_key_path" : "/path/to/private_key"
-          }
-        }
-        // Optionally, add the REST plugin
-        rest: { http_port: 8000 }
-      }
-    }
-    ```
-  - Run the zenoh router with:
-    `zenohd -c zenoh.json5`
+   Expected output is like:
 
-## Adding Typescript to your application
+   ```txt
+   zenohd: zenohd vc764bf9b built with rustc 1.75.0 (82e1608df 2023-12-21)
+   zenoh::net::runtime: Using ZID: bb3fb16628f57e92f92accf2f5c81511
+   zenoh::api::loader: Loading  plugin "remote_api"
+   zenoh::api::loader: Starting  plugin "remote_api"
+   zenoh::api::loader: Successfully started plugin remote_api from "./target/debug\\zenoh_plugin_remote_api.dll"
+   zenoh::api::loader: Finished loading plugins
+   zenoh::net::runtime::orchestrator: Zenoh can be reached at: tcp/...
+   ```
 
-The TypeScript library can be install from the command line: 
+### Building the library
 
-`npm install @eclipse-zenoh/zenoh-ts@0.0.8`
+1. Make sure that the following utilities are available on your platform.
 
-Or added via package.json
+- [NPM](https://www.npmjs.com/package/npm)
+- [yarn](https://classic.yarnpkg.com/lang/en/docs/install/#debian-stable)
+- [Typescript](https://www.typescriptlang.org/download/) 
 
-`"@eclipse-zenoh/zenoh-ts": "0.0.8" `
+2. Navigate to the directory `zenoh-ts`
+
+1. Run the commands:
+
+   ```sh
+   yarn install 
+   yarn run build
+   ```
 
-Note: In order to add this library to your project you must log into the github npm repository,  
-please refer to this link for more information [Accessing github NPM](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry#authenticating-with-a-personal-access-token)
+### Build and run the command line examples
 
+This library is currently compatible with browsers, but not with NodeJS due to websocket library limitations.
+To run the command line examples use the javascript runtime [deno](https://deno.com/) which is expected to be consistent with the browser.
 
-[zenoh]: https://github.com/eclipse-zenoh/zenoh
+1. Install [deno](https://deno.com/)
+1. Navigate to the `zenoh-ts/examples/deno` directory
+1. Install the `zenoh-ts` library by running `yarn install`
+1. Run zenohd with the remote_api plugin, configured to websocket port 10000, as described above
+1. Run the examples by running `yarn example <PATH TO EXAMPLE>`, i.e. `yarn example src/z_sub.ts`
```

Comment:
```suggestion
1. Install [deno](https://deno.com/)
2. Navigate to the `zenoh-ts/examples/deno` directory
3. Install the `zenoh-ts` library by running `yarn install`
4. Run zenohd with the remote_api plugin, configured to websocket port 10000, as described above
5. Run the examples by running `yarn example <PATH TO EXAMPLE>`, i.e. `yarn example src/z_sub.ts`
```

Reply:
same thing. Though yes, I agree that if user read this with e.g. vim, better to make correct numberring. Fixed